api/v1alpha3/awscluster_webhook_test.go

```diff
@@ -107,3 +108,48 @@ func TestAWSCluster_ValidateUpdate(t *testing.T) {
 		})
 	}
 }
+
+func TestAWSCluster_Default(t *testing.T) {
+	g := NewWithT(t)
+	tests := []struct {
+		name          string
+		beforeCluster *AWSCluster
+		afterCluster  *AWSCluster
+	}{
+		{
+			name: "CNI ingressRules are updated if it is empty",
+			beforeCluster: &AWSCluster{
+				Spec: AWSClusterSpec{},
+			},
+			afterCluster: &AWSCluster{
+				Spec: AWSClusterSpec{
+					NetworkSpec: NetworkSpec{
+						CNI: CNISpec{
+							IngressRules: IngressRules{
+								{
+									Description: "bgp (calico)",
+									Protocol:    SecurityGroupProtocolTCP,
+									FromPort:    179,
+									ToPort:      179,
+								},
+								{
+									Description: "IP-in-IP (calico)",
+									Protocol:    SecurityGroupProtocolIPinIP,
+									FromPort:    -1,
+									ToPort:      65535,
+								},
+							},
+						},
+					},
+				},
+			},
+		},
```

Comment:
Please add a test to validate that we don't add Calico when we have other ingress rules defined.